build/lib/stylelint/vscode-known-variables.json

```diff
@@ -734,6 +734,7 @@
 		"--tab-sizing-current-width",
 		"--tab-sizing-fixed-min-width",
 		"--tab-sizing-fixed-max-width",
+		"--title-height",
```

Comment:
This should probably be a bit more specific, such as `editor-group-title-height` because `title-height` is quite generic.